language/v2/language_entities_gcs.py

```diff
@@ -0,0 +1,113 @@
+#   usage: python3 samples/v2/language_entities_gcs.py [--gcs_content_uri "gs://cloud-samples-data/language/entity.txt"]
+
+# [START language_entities_gcs]
+from google.cloud import language_v2
+
+
+def sample_analyze_entities(gcs_content_uri: str) -> None:
+    """
+    Analyzes Entities in text file stored in Cloud Storage.
+
+    Args:
+      gcs_content_uri: Google Cloud Storage URI where the file content is located.
+      e.g. gs://[Your Bucket]/[Path to File]
+    """
+
+    client = language_v2.LanguageServiceClient()
+
+    # Available types: PLAIN_TEXT, HTML
+    document_type_in_plain_text = language_v2.Document.Type.PLAIN_TEXT
+
+    # Optional. If not specified, the language is automatically detected.
+    # For list of supported languages:
+    # https://cloud.google.com/natural-language/docs/languages
+    language_code = "en"
+    document = {
+        "gcs_content_uri": gcs_content_uri,
+        "type_": document_type_in_plain_text,
+        "language_code": language_code,
+    }
+
+    # Available values: NONE, UTF8, UTF16, UTF32.
+    # See https://cloud.google.com/natural-language/docs/reference/rest/v2/EncodingType.
+    encoding_type = language_v2.EncodingType.UTF8
+
+    response = client.analyze_entities(
+        request={"document": document, "encoding_type": encoding_type}
+    )
+
+    for entity in response.entities:
+        print(f"Representative name for the entity: {entity.name}")
+
+        # Get entity type, e.g. PERSON, LOCATION, ADDRESS, NUMBER, et al.
+        # See https://cloud.google.com/natural-language/docs/reference/rest/v2/Entity#type.
+        print(f"Entity type: {language_v2.Entity.Type(entity.type_).name}")
+
+        # Loop over the metadata associated with entity.
+        # Some entity types may have additional metadata, e.g. ADDRESS entities
+        # may have metadata for the address street_name, postal_code, et al.
+        for metadata_name, metadata_value in entity.metadata.items():
+            print(f"{metadata_name}: {metadata_value}")
+
+        # Loop over the mentions of this entity in the input document.
+        # The API currently supports proper noun mentions.
+        for mention in entity.mentions:
+            print(f"Mention text: {mention.text.content}")
+
+            # Get the mention type, e.g. PROPER for proper noun
+            print(
+                'Mention type:'
+                f' {language_v2.EntityMention.Type(mention.type_).name}'
+            )
+
+            # Get the probability score associated with the first mention of the entity in the (0, 1.0] range.
+            print(f"Probability score: {mention.probability}")
+
+    # Get the language of the text, which will be the same as
+    # the language specified in the request or, if not specified,
+    # the automatically-detected language.
+    print(f"Language of the text: {response.language_code}")
+
+
+# [END language_entities_gcs]
+
+
+def main():
+    import argparse
```

Comment:
nit: consider adding the import before the region tag, around line 25 so imports are all in one place - but I'm not 100% sure if the linter would be okay with that. Fine to leave it here for now since this is a more local import.

Reply:
Thanks!